src/app/components/CallToActionLinkComponent/Text/README.md

```diff
@@ -0,0 +1,82 @@
+# Call To Action Link Text
+
+## Description
+
+A component that uses the Text component. It can be used to apply sets of custom styles.
+
+This component derives `size` and `fontVariant` values from the Context Provider. If none are provided it falls back to default values (`size`: `pica`, `fontVariant`: `sansBold`).
+
+## Props
+
+| Name                         | type    | Description                                                                           |
+| ---------------------------- | ------- | ------------------------------------------------------------------------------------- |
+| as?                          | string  | The HTML element to render the text in e.g. `h1`, `span`, `p`. The default is `span`. |
```

Comment:
Shall we remove the `?` here, and instead have a column for whether it's mandatory or optional?